packages/client/src/pages/main/modules/mainSlide/MainSlide.tsx

```diff
@@ -19,11 +19,13 @@ const {
 const MainSlide = () => {
   const [searchParams] = useSearchParams()
   const [signInWithOauth] = useOauthMutation()
+  const navigate = useNavigate()
 
   useEffect(() => {
     const code = searchParams.get(`code`)
     if (!code) return
     signInWithOauth(code)
+    return navigate('/game')
```

Comment:
Лучше ничего не хардкодить таким образом, если роут вдруг поменяется, то вариант ниже подтянет всё автоматом
 
```suggestion
    return navigate(GAME_ROUTE)
```